static/docs.css

```diff
@@ -108,7 +108,7 @@ a.sc-logo img {
   font-size: 16px;
 }
 
-.side-nav .nav-header a, .side-nav .nav-header span {
```

Comment:
Yep, I was worried about that too and checked - all good even if I delete your JS. Everything just is expanded then, but looks correct.